V2/gameFuseUtilities.js

```diff
@@ -4,29 +4,28 @@
             if (!success){
                 callback(responseMessage,true)
             }
-            else if (response == undefined) {
-
+            else if (response == null && callback != null) {
                 callback(responseMessage, true)
             }
             else if (response.status >= 400) {
                 console.error(`Request (${response.url}) had error: ` + response.data);
-                if (callback !== null)
+                if (callback != null)
```

Comment:
`!= null` also checks for undefined. `undefined == null` returns true, whereas `undefined === null` does not. So doing `== null` or `!= null` is good because it checks for both.

Reply:
nice catch thanks